src/models/CloudsDetails.tsx

```diff
@@ -0,0 +1,11 @@
+export class CloudsDetails {
```

Comment:
What's the purpose of this class?

Reply:
I was thinking that maybe we could just use { all: number } instead of this class.